src/games/strategy/engine/framework/GameRunner2.java

```diff
@@ -137,16 +138,15 @@ private static void usage() {
   public static void main(final String[] args) {
     ErrorConsole.getConsole();
 
-    try (PerfTimer timer = Perf.startTimer("Show main window")) {
-      // do after we handle command line args
-      checkForMemoryXMX();
-      SwingUtilities.invokeLater(() -> setupLookAndFeel());
-      showMainFrame();
-    }
-    (new Thread(() -> setupLogging())).start();
-
+    PerfTimer timer = PerfTimer.startTimer("Show main window");
+    // do after we handle command line args
+    checkForMemoryXMX();
+    SwingUtilities.invokeLater(() -> setupLookAndFeel());
+    showMainFrame();
+    new Thread(() -> setupLogging()).start();
+    timer.stop();
```

Comment:
In addition, the timer is being changed, now includes the time to spawn a new thread. Not consequential, but we should be careful of functionality changes. 
Bottom line though, the timer is more useful I think for spot checking timings, rather than for the metrics reporting it was originally intended for, so the main window timing can be removed altogether.